piker/brokers/kraken.py

```diff
@@ -523,50 +538,65 @@ async def handle_order_requests(
             # validate
             temp_id = next(userref_counter)
             order = BrokerdOrder(**request_msg)
+            def slashinsert(str):
+                midPoint = len(str)//2
+                return str[:midPoint] + '/' + str[midPoint:]
+            # Send order via websocket
+            order_msg = {
+                          "event": "addOrder",
+                          "ordertype": "limit",
+                          "pair": slashinsert(order.symbol.upper()),
+                          "price": str(order.price),
+                          "token": token,
+                          "type": order.action,
+                          "volume": str(order.size)
+                        }
+                    
+
+            await ws.send_msg(order_msg)
```

Comment:
Ok so i guess from kraken docs you're supposed to get pub messages back on the `ws` of the form:

```json
{
  "descr": "buy 0.01770000 XBTUSD @ limit 4000",
  "event": "addOrderStatus",
  "status": "ok",
  "txid": "ONPNXH-KMKMU-F4MR5V"
}
```
i'm seeing from [here](https://docs.kraken.com/websockets/#message-addOrder).

I'll try testing this in a bit.

Reply:
yeah these should also show up in the `process_order_msgs()` handler below *i think?*

Reply:
these dont come in if you are subscribed. Thats the response you would get if you just made the request. You get a version of that information in the format they line out in the description of the stream. The docs are really poorly written in that sense.